x/distribution/keeper/delegation_test.go

```diff
@@ -270,28 +270,31 @@ func TestWithdrawDelegationRewardsBasic(t *testing.T) {
 	power := int64(100)
 	valTokens := staking.TokensFromTendermintPower(power)
 	commission := staking.NewCommissionMsg(sdk.NewDecWithPrec(5, 1), sdk.NewDecWithPrec(5, 1), sdk.NewDec(0))
-	msg := staking.NewMsgCreateValidator(valOpAddr1, valConsPk1,
-		sdk.NewCoin(staking.DefaultBondDenom, valTokens), staking.Description{}, commission, sdk.OneInt())
+	msg := staking.NewMsgCreateValidator(
+		valOpAddr1, valConsPk1,
+		sdk.NewCoin(staking.DefaultBondDenom, valTokens),
+		staking.Description{}, commission, sdk.OneInt(),
+	)
 	require.True(t, sh(ctx, msg).IsOK())
 
 	// assert correct initial balance
 	expTokens := balanceTokens.Sub(valTokens)
 	require.Equal(t,
-		sdk.Coins{{staking.DefaultBondDenom, expTokens}},
-		ak.GetAccount(ctx, sdk.AccAddress(valOpAddr1)).GetCoins())
+		sdk.Coins{sdk.NewCoin(staking.DefaultBondDenom, expTokens)},
+		ak.GetAccount(ctx, sdk.AccAddress(valOpAddr1)).GetCoins(),
+	)
 
 	// end block to bond validator
 	staking.EndBlocker(ctx, sk)
 
-	// set zero outstanding rewards
-	k.SetOutstandingRewards(ctx, sdk.DecCoins{})
-
 	// fetch validator and delegation
 	val := sk.Validator(ctx, valOpAddr1)
 
 	// allocate some rewards
 	initial := staking.TokensFromTendermintPower(10)
-	tokens := sdk.DecCoins{{staking.DefaultBondDenom, sdk.NewDecFromInt(initial)}}
+	tokens := sdk.DecCoins{sdk.NewDecCoin(staking.DefaultBondDenom, initial)}
+
+	k.SetOutstandingRewards(ctx, tokens)
```

Comment:
note this change @cwgoes -- we cannot have zero initial outstanding rewards as that would result in a transient negative balance during `WithdrawDelegationRewards`.

Reply:
great, thanks